ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.tsx

```diff
@@ -1,16 +1,75 @@
+import { useEffect } from "react";
+
+import { useSelector, useDispatch } from "react-redux";
+
 import DiscoveriesListHeader from "../DiscoveriesListHeader";
 
 import Section from "app/base/components/Section";
 import { useWindowTitle } from "app/base/hooks";
+import { actions } from "app/store/discovery";
+import discoverySelectors from "app/store/discovery/selectors";
 
 const DiscoveriesList = (): JSX.Element => {
   useWindowTitle("Dashboard");
 
+  const discoveries = useSelector(discoverySelectors.all);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(actions.fetch());
+  }, [dispatch]);
+
   return (
     <Section
       header={<DiscoveriesListHeader />}
       headerClassName="u-no-padding--bottom"
-    />
+    >
+      <table>
+        <thead>
+          <tr>
+            <th style={{ width: "20%" }}>Name</th>
+            <th style={{ width: "15%" }}>Mac address</th>
+            <th style={{ width: "15%" }}>IP</th>
+            <th style={{ width: "15%" }}>Rack</th>
+            <th>Last seen</th>
+            <th className="u-align--right" style={{ width: "75px" }}>
+              Action
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {discoveries
```

Comment:
The `discoverySelectors.all` selector should always return an array so I think this ternary might be unnecessary.